conf/messages

```diff
@@ -131,7 +131,7 @@ agentHub.changeClient = Change client
 breadcrumb.agentServices = Agent Services Account
 
 signUpPartial.LinkText = Sign up for Making Tax Digital for VAT
-signUpPartial.Body = If your client’s taxable turnover exceeds the VAT threshold, they must sign up to Making Tax Digital for VAT.
+signUpPartial.Body = Your client must sign up to Making Tax Digital for VAT if they are not exempt from VAT and their taxable turnover exceeds the £85,000 threshold.
```

Comment:
You've just missed a comma out between ".. from VAT" and "and their...", apart from that all looks good